### DISCUSSION TITLE
Next 13 overflow-x showing on page load/change but when you scroll it's gone?

### DISCUSSION THREAD

**Discussion by Galanthus** [vercel/next.js#57492] [Help]
### Summary

Problem where when switching between routes I get this weird behaviour where there is a overflow-x (left & right) but when I scroll it's gone. Setting overflow hidden on the HTML tag doesn't do anything.

**Comment by Hillsie** [comment]
If you're using tailwindcss, you could try adding `overflow-clip` on the main in your root `layout`.  Content that extends outside the box can be a challenge to find.  My debugging approach is to add some background color with opacity or add borders on components until I find the culprit outside the box. Clipping it can be the easier solution, though.
